apps/climatemappedafrica/src/pages/[[...slug]].js

```diff
@@ -11,9 +13,24 @@ const componentsBySlugs = {
   team: AboutTeam,
 };
 
-function Index({ blocks, fallback, ...props }) {
-  if (!blocks?.length) {
-    return null;
+function Index({ blocks, menus, footer: footerProps, seo = {}, fallback }) {
+  const pageSeo = {};
+  pageSeo.title = seo?.title || undefined;
```

Comment:
```suggestion
  pageSeo.title = seo?.title || null;
```

`seo?.title` is already `undefined` if `seo` is `undefined`. Why would we need the `extra || undefined`?